lib/common/collections/wp_items/detectable.rb

```diff
@@ -31,7 +31,7 @@ def aggressive_detection(wp_target, options = {})
         request.on_complete do |response|
           progress_bar.progress += 1 if options[:show_progression]
 
-          if target_item.exists?(exist_options, response)
+          if target_item.exists?(exist_options, response) or (target_item.class.to_s == 'WpUser' and options[:display_name])

```

Comment:
Because this is the first time I use Ruby (and I never studied it before, but I will).
By the way, I'll replace it with a `require` and the `kind_of` method. Is it the right way to do it?